packages/astro/client.d.ts

```diff
@@ -1,5 +1,11 @@
 /// <reference types="vite/types/importMeta" />
 
+// HTML
```

Comment:
Can you import those from non-Astro files? If not, this type should be in `astro/env`